app/src/main.tsx

```diff
@@ -148,9 +148,10 @@ function HuuApp() {
         <Route
           path="/profile/:profileId"
           element={
-            <ProtectedRoute>
-              <IntakeProfile />
-            </ProtectedRoute>
+            // <ProtectedRoute>
+            //   <IntakeProfile />
+            // </ProtectedRoute>
+            <IntakeProfile />
```

Comment:
Please uncomment the protected route.

Reply:
Thanks Erik, I removed the commented protected route.